runtime/src/bank.rs

```diff
@@ -2843,22 +2845,7 @@ impl Bank {
     }
 
     pub fn calculate_capitalization(&self) -> u64 {
-        self.get_program_accounts(None)
-            .into_iter()
-            .map(|(_pubkey, account)| {
-                let is_specially_retained = solana_sdk::native_loader::check_id(&account.owner)
-                    || solana_sdk::sysvar::check_id(&account.owner);
-
-                if is_specially_retained {
-                    // specially retained accounts are ensured to exist by
-                    // always having a balance of 1 lamports, which is
-                    // outside the capitalization calculation.
-                    account.lamports - 1
-                } else {
-                    account.lamports
-                }
-            })
-            .sum()
```

Comment:
https://github.com/solana-labs/solana/pull/11927#issuecomment-684114178

> btw, because the testnet ledger was restarted, it's now safe to start to enforce this everytime validator starts. (thanks @mvines).

very sadly, `sum()` was so subtle. my checked math in-brain type checker got tricked. ;) **This does with unchecked math.** So, testnet capitalization was still off-calcualtion combined with pre-mature removal of storage account special casing at #9992.

I'll first have to create another gated pr to really remove rewards_pool in testnet.

Reply:
> I'll first have to create another gated pr to really remove rewards_pool in testnet.

This is done #12011